R/preprocess_data.R

```diff
@@ -52,19 +49,37 @@ preprocess_data <- function(dataset, outcome_colname, method = c("center", "scal
 
   # process nonbinary features
   cont_feats_transformed <- cont_feats
+  removed_cont <- NULL
   if (!is.null(cont_feats_transformed)) {
-    cont_feats_transformed <- process_cont_feats(cont_feats, method)
+    feats <- process_cont_feats(cont_feats, method)
+    cont_feats_transformed <- feats$transformed_cont
+    removed_cont <- feats$removed_cont
   }
 
   # combine all features
   processed_feats <- dplyr::bind_cols(cont_feats_transformed, cat_feats, novar_feats)
 
   # remove features with non-zero variance
-  if (remove_nzv) processed_feats <- get_caret_processed_df(processed_feats, "nzv")
+  removed_feats <- removed_cont
```

Comment:
Why do this instead of continuing with the `removed_cont` variable name?

Reply:
oops no reason. just changed it